pkg/apis/pipeline/v1alpha1/git_resource.go

```diff
@@ -30,7 +42,8 @@ type GitResource struct {
 	// Git revision (branch, tag, commit SHA or ref) to clone.  See
 	// https://git-scm.com/docs/gitrevisions#_specifying_revisions for more
 	// information.
-	Revision string `json:"revision"`
+	Revision   string `json:"revision"`
+	TargetPath string
 }
 
 // NewGitResource create a new git resource to pass to Knative Build
```

Comment:
do we want to support upload for git?

Reply:
Yes that would be awesome. I don't know if I want to do that feature in this PR.